csharp/BUILD

```diff
@@ -52,9 +51,96 @@ csharp_library(
         "//csharp/Connection:connection",
         "//csharp:typedb_driver_pinvoke",
     ],
-    visibility = ["//visibility:public"],
+    out = "TypeDB.Driver",
     target_frameworks = target_frameworks,
     targeting_packs = targeting_packs,
+    visibility = ["//visibility:public"],
+)
+
+
+swig_native_nuget_pack(
```

Comment:
Produces a set of native libs, picks one for our current platform to push further.